src/System.Runtime.Serialization.Xml/tests/SerializationTypes.cs

```diff
@@ -3202,3 +3202,102 @@ public class CollectionOfTypeWithNonDefaultNamcespace : List<TypeWithNonDefaultN
 
 }
 
+[DataContract]
+public class ObjectContainer
+{
+    [DataMember]
+    private object data;
+
+    [DataMember]
+    private object data2;
+
```

Comment:
Question only: what's the difference between `data` and `data2`? Why do we need two fields?